test/calculator_test.js

```diff
@@ -0,0 +1,44 @@
+const assert = require('assert');
+const chai = require('chai');
+const calc = require('../calculator')
+
+describe('calculator', function () {
```

Comment:
These tests are an amazing start! 😍 Would it be worth having some tests for the other operators? 😀